src/styles/abstracts/_mixins.scss

```diff
@@ -38,3 +38,244 @@
     text-overflow: ellipsis;
     white-space: nowrap;
 }
+
+@mixin content-padding($line-height) {
```

Comment:
this mixin makes me nervious. why is it creating content padding based on line height, and resetting margins within this seems like a potential unintended side effect. Additionally, this only sets a padding on top and bottom and sets the sides to zero...

Reply:
We'll discuss this, and fix it up together.

Reply:
Updating to only set the top and bottom. Octuple's current visual design spec is 25% the line height of the typeface in any given selector.  Horizontal padding is deferred to the grid.

Reply:
Scoped and resolved.